Scoop/Controller/Onboarding/OnboardingViewController.swift

```diff
@@ -9,21 +9,23 @@ import UIKit
 
 protocol OnboardingDelegate: AnyObject {
     func didTapNext(_ viewController: UIViewController, nextViewController: UIViewController?)
+    func didTapBack(_ viewController: UIViewController, previousViewController: UIViewController?)
 }
 
 class OnboardingViewController: UIViewController {
     
     internal weak var delegate: OnboardingDelegate?
     internal var nextAction: UIAction?
```

Comment:
Can this be private?

Reply:
I tried making it private, but it ended up being inaccessible to the rest of the view controllers, so I'm not sure I could set it private